mifosng-provider/src/integrationTest/java/org/mifosplatform/integrationtests/CenterIntegrationTest.java

```diff
@@ -181,4 +183,51 @@ public void testCenterDeletion() {
         }
         return groupMembers;
     }
+
+    @Test
+    public void testStaffAssignmentDuringCenterCreation() {
+
+        Integer staffId = StaffHelper.createStaff(this.requestSpec, this.responseSpec);
+        System.out.println("--------------creating first staff with id-------------" + staffId);
+        Assert.assertNotNull(staffId);
+
+        Integer centerID = CenterHelper.createCenterWithStaffId(this.requestSpec, this.responseSpec, staffId);
+        CenterHelper.verifyCenterCreatedOnServer(this.requestSpec, this.responseSpec, centerID);
+
+    }
+
+    @Test
+    public void testAssignStaffToCenter() {
+        Integer staffId = StaffHelper.createStaff(this.requestSpec, this.responseSpec);
+        System.out.println("--------------creating first staff with id-------------" + staffId);
+        Assert.assertNotNull(staffId);
+
+        Integer groupID = CenterHelper.createCenter(this.requestSpec, this.responseSpec);
+        CenterHelper.verifyCenterCreatedOnServer(this.requestSpec, this.responseSpec, groupID);
+
+        HashMap assignStaffGroupId = (HashMap) CenterHelper.assignStaff(this.requestSpec, this.responseSpec, groupID.toString(),
+                staffId.longValue());
+        assertEquals("Verify assigned staff id is the same as id sent", assignStaffGroupId.get("staffId"), staffId);
```

Comment:
for verification, get client by Id and ensure the assigned staff has the same Id as the staff If used in the previous association request